CONTRIBUTING

```diff
@@ -0,0 +1,39 @@
+Developer Certificate of Origin
```

Comment:
Maybe hold off on this one for now -- we're still figuring out if we _need_ it in this form (the DCO implies that everyone uses `git commit -s`).